pkgs/development/libraries/libmatheval/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, guile, autoconf, flex }:
+
+stdenv.mkDerivation rec {
+  version = "1.1.11";
+  name = "libmatheval-${version}";
+
+  buildInputs = [ guile autoconf flex ];
```

Comment:
I think at least `autoconf` and `flex` could be moved to `nativeBuildInputs`, if they are not used at run-time somehow.